pages/icon/size-inherit.page.tsx

```diff
@@ -15,6 +16,7 @@ export default function IconScenario() {
   const [height, setHeight] = useState(16);
   const [visibility, setVisibility] = useState(false);
   const unitHeight = height + UNITS;
+  const [iconName, setIconName] = useState('settings' as IconProps.Name);
```

Comment:
Also, I would recommend using url params instead of state, as it shown here: https://github.com/cloudscape-design/components/blob/d48075d6af501105cc5ad6b7e9e4a16b400ab142/pages/content-layout/without-header.page.tsx#L20

This will come in handy when you write screenshot tests, because you can open the desired icon by the direct url